lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -3045,6 +3056,20 @@ void WireOp::getAsmResultNames(OpAsmSetValueNameFn setNameFn) {
 
 std::optional<size_t> WireOp::getTargetResultIndex() { return 0; }
 
+LogicalResult WireOp::verifySymbolUses(SymbolTableCollection &symbolTable) {
+  auto circuitOp = getOperation()->getParentOfType<CircuitOp>();
+  auto refType = dyn_cast<RefType>(getType(0));
+  if (!refType)
+    return success();
+
+  auto layer = refType.getLayer();
+  if (layer && !symbolTable.lookupSymbolIn(circuitOp, layer))
```

Comment:
```suggestion
  if (layer && !symbolTable.lookupSymbolIn<LayerOp>(circuitOp, layer))
```
?
(to match the other)

Reply:
Fixed in https://github.com/llvm/circt/pull/6574/commits/68b4e01b15af105b51776e0225616d1cf8853e38.